src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh

```diff
@@ -17,23 +16,32 @@ df 1>&2
 cat /proc/meminfo 1>&2
 
 start_time=$(date +%s)
-build_id=$(date +%Y%m%d-%H%M)
 
 aws configure set region $AWS_REGION
 
+if [ ! -z "${BOTO_ENDPOINT_URL}" ]; then
```

Comment:
The changes to this file are ~identical to the changes to the ondemand script, they just make the two scripts more similar.